### ISSUE TITLE
Script fails on grep command

### ISSUE THREAD

**Issue by Techtony96** [ianharrier/synology-scripts#20]
```
Task Scheduler has completed a scheduled task.

Task: Reconnect VPN
Start time: Fri, 24 Jun 2022 23:00:01 GMT
Stop time: Fri, 24 Jun 2022 23:01:01 GMT
Current status: 2 (Interrupted)
Standard output/error:
[I] There is 1 VPN profile. Continuing...
[I] Synology DSM reports VPN is connected.
Usage: grep [OPTION]... PATTERN [FILE]...
Try 'grep --help' for more information.
[W] The gateway IP did not respond to ping.
[I] Attempting to reconnect...

kill client ...OK
get arguemnt id: o1634173141
[I] Synology DSM reports VPN is connected.
Usage: grep [OPTION]... PATTERN [FILE]...
Try 'grep --help' for more information.
[W] The gateway IP did not respond to ping.
[E] VPN failed to reconnect. Exiting...
```

**Comment by melbnz** [comment]
I have exactly the same issue - started a 9 June after i updated to 7.1 I think - the synology log is unhelpful.  I can see a number of packages being repaired/updated and the 6.x UI being uninstalled in log
i can ping VPN gw using SSH - though did have to chmod ping permission, and i get the same grep error emailed to me.

**Comment by melbnz** [comment]
After reading the other open issues i have resolved this by following: issue after updating to Synology DSM 7.1 #16 
Edit the script TUNNEL_INTERFACE=$(ip addr | grep $CLIENT_IP | awk '{ print $7 }')  to be $5 not $7
Thanks clever people :-)

**Comment by epposan** [comment]
I have systems running the newer DSM 7.1 and the older DSM 7.0 (no upgrade offered yet).
Instead of changing **$7** into **$5**, I suggest changing **$7** into **$NF**, making awk to print the last parameter, which seems to be the one we want. That way the script remains identical for all versions.

**Comment by ianharrier** [comment]
This is fixed in version 1.3.1. Thanks for reporting!
